examples/documentation/PTI_experiment/PTI_Experiment_Recon3D_anisotropic_target_small.py

```diff
@@ -0,0 +1,677 @@
+]
+mean_permittivity_PT = np.array(
+    [
+        ((-1) ** i)
+        * util.wavelet_softThreshold(
+            ((-1) ** i) * mean_permittivity_PT[i], "db8", 0.00303, level=1
+        )
+        for i in range(2)
+    ]
+)
+
+
+# %%
+# Visualize reconstructed physical properties of the anisotropic glass target
+
+# %%
+# Reconstructed phase, absorption, mean permittivity, azimuth, and inclination assuming (+) and (-) optic sign
+
+# %%
+# browse the reconstructed physical properties
+visual.parallel_4D_viewer(
+    np.transpose(
+        np.stack(
+            [
+                phase_PT,
+                mean_permittivity_PT[0],
+                azimuth[0],
+                theta[0],
+                absorption_PT,
+                mean_permittivity_PT[1],
+                azimuth[1],
+                theta[1],
+            ]
+        ),
+        (3, 0, 1, 2),
+    ),
+    num_col=4,
+    origin="lower",
+    set_title=True,
+    titles=[
+        r"phase",
+        r"mean permittivity (+)",
+        r"$\omega$ (+)",
+        r"$\theta$ (+)",
+        r"absorption",
+        r"mean permittivity (-)",
+        r"$\omega$ (-)",
+        r"$\theta$ (-)",
+    ],
+)
+
+
+# %%
+# Load the processed results
+PTI_file_name = (
+    "/path/to/Anisotropic_target_small/Anisotropic_target_small_processed.zarr"
+)
+reader = zarr.open(PTI_file_name, mode="r")
+PTI_array = np.array(reader["Row_0/Col_0/f_tensor/array"])
+print(PTI_array.shape)
+PTI_array = np.transpose(PTI_array, (0, 1, 3, 4, 2))[0]
+f_tensor = PTI_array[:7]
+mat_map = PTI_array[7:]
+
+# compute the physical properties from the scattering potential tensor
+
+mean_permittivity_p, azimuth_p, theta_p = (
+    optics.scattering_potential_tensor_to_3D_orientation_PN(
+        f_tensor, material_type="positive", reg_ret_pr=reg_mean_permittivity
+    )
+)
+mean_permittivity_n, azimuth_n, theta_n = (
+    optics.scattering_potential_tensor_to_3D_orientation_PN(
+        f_tensor, material_type="negative", reg_ret_pr=reg_mean_permittivity
+    )
+)
+mean_permittivity = np.array([mean_permittivity_p, mean_permittivity_n])
+azimuth = np.array([azimuth_p, azimuth_n])
+theta = np.array([theta_p, theta_n])
+
+p_mat_map = optics.optic_sign_probability(mat_map, mat_map_thres=0.09)
+phase = optics.phase_inc_correction(
+    f_tensor[0], mean_permittivity[1], theta[1]
+)
+phase_PT, absorption_PT, mean_permittivity_PT = [
+    optics.unit_conversion_from_scattering_potential_to_permittivity(
+        SP_array, lambda_illu, n_media=n_media, imaging_mode="3D"
+    )
+    for SP_array in [phase, f_tensor[1].copy(), mean_permittivity]
+]
+mean_permittivity_PT = np.array(
+    [
+        ((-1) ** i)
+        * util.wavelet_softThreshold(
+            ((-1) ** i) * mean_permittivity_PT[i], "db8", 0.00303, level=1
+        )
+        for i in range(2)
+    ]
+)
+
+
+# %%
+# Phase, mean permittivity, azimuth, inclination, and optic sign reconstruction
+
```

Comment:
@talonchandler looks like you read that mean permittivity is equivalent to principal retardance. It is actually the differential permittivity. I was partial to isotropic permittivity and anisotropic permittivity, but this is the terminology we have now throughout the figures and videos. Can you check the variable names in previous section and this section?